components/cam/src/physics/cam/clubb_intr.F90

```diff
@@ -469,9 +480,12 @@ subroutine clubb_readnl(nlfile)
       call mpibcast(clubb_tk2,                1,   mpir8,   0, mpicom)
       call mpibcast(relvar_fix,               1,   mpilog,  0, mpicom)
       call mpibcast(clubb_use_sgv,            1,   mpilog,   0, mpicom)
+      call mpibcast(clubb_vert_avg_closure,   1,   mpilog,   0, mpicom)
+      call mpibcast(clubb_ipdf_call_placement,   1,   mpilog,   0, mpicom)
```

Comment:
change mpilog to mpiint in 
`call mpibcast(clubb_ipdf_call_placement,   1,   mpilog,   0, mpicom)`

Reply:
Thanks for identifying the data type issue. I already changed and committed it to the branch